sdcm/provision/common/provision_plan.py

```diff
@@ -0,0 +1,34 @@
+import logging
```

Comment:
Evgeniy would ask for the licence comment block.

Reply:
added